Makefile

```diff
@@ -453,6 +474,29 @@ endif
 yosys-abc$(EXE): abc/abc-$(ABCREV)$(EXE)
 	$(P) cp abc/abc-$(ABCREV)$(EXE) yosys-abc$(EXE)
 
+abc/libabc-$(ABCREV).a:
```

Comment:
This seems to be only different from `abc/abc-$(ABCREV)$(EXE)` above in the make target passed to ABC. I think this rule can be merged with the rule for `abc/abc-$(ABCREV)$(EXE)`.

Something like the following might work (warning: untested nontrivial Makefile code ahead :).

```
abc/abc-$(ABCREV)$(EXE) abc/libabc-$(ABCREV).a:
    ....
    $(Q) cd abc && $(MAKE) ... $(if $(filter %.a,$@),libabc-$(ABCREV).a)
```